src/features/CourtCaseDetails/CourtCaseDetails.stories.tsx

```diff
@@ -63,3 +63,22 @@ DetailsLockedByAnotherUser.play = ({ canvasElement }) => {
   ).toHaveLength(2)
   expect(canvas.queryByText("Add Note")).not.toBeInTheDocument()
 }
+
+export const TriggersNotVisibleToUser: ComponentStory<typeof CourtCaseDetails> = () => (
+  <CourtCaseDetails courtCase={courtCase} lockedByAnotherUser={false} aho={aho} triggersVisible={false} />
+)
+
+TriggersNotVisibleToUser.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement)
+  await expect(canvas.getByText(courtCase.ptiurn)).toBeInTheDocument()
+  await expect(canvas.getByText(courtCase.courtName)).toBeInTheDocument()
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
iirc I turned this eslint error off so I don't think we need this anymore

Reply:
Good spot. 👍